astronify/series/series.py

```diff
@@ -295,6 +295,99 @@ def play(self):
             delay=list(delays), dur=duration
         )
 
+    def play_notebook(self):
```

Comment:
Should this verb (function) be "play_in_notebook" instead?

Reply:
@scfleming: I updated my branch. Any other changes you need to merge this?